integrations/cdi/jpa-cdi/src/main/java/io/helidon/integrations/cdi/jpa/Extended.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.cdi.jpa;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.enterprise.util.AnnotationLiteral;
+import javax.inject.Qualifier;
+
+/**
+ * A {@link Qualifier} indicating that the qualified bean's instances
+ * are associated with a JTA transaction.
+ *
+ * <p>This qualifier must not be combined with {@link
+ * CdiTransactionScoped}, {@link JpaTransactionScoped} or {@link
+ * NonTransactional}.</p>
+ */
+@Qualifier
+@Retention(RetentionPolicy.RUNTIME)
+@Target({}) // can only be programmatically added
+@interface Extended {
```

Comment:
Name seems too generic; based on the class description, shouldn't it have "transaction" in the name?

Reply:
Oh, just noticed the `@Target` comment. If this is an implementation detail, not exposed to users, then the name is obviously less relevant.

Reply:
For the record, I think "extended" is awful, but it is standard JPA-speak.  An extended persistence context is one that spans transactions.  My pattern here was to use the [short name of the analogous Java EE/JPA annotation](https://javaee.github.io/javaee-spec/javadocs/javax/persistence/PersistenceContextType.html#EXTENDED).  And you are correct this is an implementation detail.

Reply:
gotcha